### PR TITLE
BBL-218 | WAF module issues and repo std structure updated

### LINKED ISSUE #10: WAF regional should support tags
### What?
- waf regional was missing "tags" (https://github.com/binbashar/terraform-aws-waf-owasp/pull/9)
- waf regional added a tags to sample (https://github.com/binbashar/terraform-aws-waf-owasp/pull/8)

### Why?
Adding tags to AWS resources brings several benefits -> https://aws.amazon.com/answers/account-management/aws-tagging-strategies/

### LINKED ISSUE #11: The example has a bad git URL
The example module has:
```
source = "git::git@github.com:binbashar/terraform-aws-owasp.git//modules/waf-regional?ref=v1.0.0"
```
but it should be:
```
source = "git::git@github.com:binbashar/terraform-aws-waf-owasp.git//modules/waf-regional?ref=v1.0.0"
```
(note the missing `-waf`)

### LINKED ISSUE #13: Upgrade to terraform-docs > v0.9.x
### What?

* :white_check_mark: Upgrade to terraform-docs `> v0.9.x` (https://github.com/segmentio/terraform-docs/releases) via dockerized approach
* :white_check_mark: Follow https://github.com/binbashar/terraform-aws-ec2-basic-layout/blob/5bfd521558eaefa7da85ed4d0a376287e1a0e7d3/Makefile#L77 as reference

### Why? 

* :heavy_check_mark: Improved documentation support for tf `0.12.x`

### PR DESCRIPTION
**Pull request by exequielrafaela**
### Commits on Aug 04, 2020
- @exequielrafaela - BBL-218 | repo updated to follow the latest std structure - cfb9927
- @exequielrafaela - BBL-218 | READMES.md have been updated with latest verions + inputs and outputs + fixed source module links - 7395fc2
- @exequielrafaela - BBL-218 | make format applied to waf-regional + updating csrf_token_set data to receive a custom HEADER as a variable - a3894e4
- @exequielrafaela - BBL-218 | make format applied to waf-global + updating csrf_token_set data to receive a custom HEADER as a variable - 686a8f9

---

- @exequielrafaela - BBL-218 | fixing CI tests - 738ae82
- @exequielrafaela - BBL-218 | fixing static code analysis and linting test - 76f92ca
- @exequielrafaela - BBL-218 | adding .editorconfig + data source for waf-regional - dd7605f
- @exequielrafaela - BBL-218 | fixing static test aws creds permissions - 213f727
- @exequielrafaela - BBL-218 | adding sudo to chown cmd - 0986230
@exequielrafaela
BBL-218 | updating waf-regional config vpc data source + adding tags to waf-global to be tested - 035db1b

### Related Issues
closes #10
closes #11
closes #13

### CODE REVIEW

File: modules/waf-regional/tests/fixture/config.tf [hunk 1]
```diff
@@ -2,8 +2,10 @@
 # AWS Provider Settings       #
 #=============================#
 provider "aws" {
-  region  = var.region
-  profile = var.profile
+  version                 = "~> 2.70"
+  region                  = var.region
+  profile                 = var.profile
+  shared_credentials_file = "~/.aws/bb/config"
```
Comment (diego-ojeda-binbash) [inline modules/waf-regional/tests/fixture/config.tf]: @exequielrafaela I think we can replace "bb" with one of the "project" variables?

File: modules/waf-global/tests/fixture/config.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#=============================#
+# AWS Provider Settings       #
+#=============================#
+provider "aws" {
+  version                 = "~> 2.70"
+  region                  = var.region
+  profile                 = var.profile
+  shared_credentials_file = "~/.aws/bb/config"
```
Comment (diego-ojeda-binbash) [inline modules/waf-global/tests/fixture/config.tf]: @exequielrafaela Same comment here about replacing "bb" with one of the "project" variables 😉

**Review by diego-ojeda-binbash — APPROVED** [review]

File: modules/waf-regional/tests/fixture/config.tf (hunk 1 above)
Comment (exequielrafaela) [reply modules/waf-regional/tests/fixture/config.tf]: @diego-ojeda-binbash OJ thanks!

File: modules/waf-global/tests/fixture/config.tf (hunk 2 above)
Comment (exequielrafaela) [reply modules/waf-global/tests/fixture/config.tf]: @diego-ojeda-binbash OJ thanks!

**Pull request merged**